src/generators/02_buying.py

```diff
@@ -3,6 +3,7 @@
 
 BUY_ITEM = "give @a[scores={gurkoins=$price..,shop_$id=1..}] $item $amount"
 BUY_EFFECT = "effect give @a[scores={gurkoins=$price..,shop_$id=1..}] $item $amount 0 true"
+BUY_COMMAND = "execute as @a[scores={gurkoins=$price..,shop_$id=1..}] in minecraft:overworld run $command"
```

Comment:
What's this for?

Reply:
Having some command effects, used by the tp_show and clear_weather options